reana_job_controller/kubernetes_job_manager.py

```diff
@@ -166,11 +170,68 @@ def execute(self):
                                 "args": [self.cmd],
                                 "name": "job",
                                 "env": [],
-                                "volumeMounts": [],
+                                "volumeMounts": [
+                                    {
+                                        "name": "applog",
+                                        "mountPath": "/opt/app",
+                                    }
+                                ],
+                                "lifecycle": {
+                                    "postStart": {
+                                        "exec": {
+                                            "command": ["/bin/sh", "-c", "touch /opt/app/running"]
+                                        }
+                                    },
+                                    "preStop": {
```

Comment:
This only fires if workflow is stopped manually. If job just completes, this will not be executed.